common/src/main/java/com/wynntils/mc/mixin/ChatComponentMixin.java

```diff
@@ -22,9 +41,151 @@ public abstract class ChatComponentMixin implements ChatComponentExtension {
     @Shadow
     private void refreshTrimmedMessages() {}
 
+    @Shadow
+    @Final
+    private List<GuiMessage.Line> trimmedMessages;
+
+    @Shadow
+    public abstract double getScale();
+
+    @Shadow
+    @Final
+    private Minecraft minecraft;
+
+    @Unique
+    private ChatTimestampFeature timestampFeature;
+
+    @Unique
+    private int timestampWidth = 0;
+
+    @WrapOperation(
+            method = "addMessageToDisplayQueue",
+            at = @At(value = "INVOKE", target = "Ljava/util/List;add(ILjava/lang/Object;)V"))
+    private void addMessageToDisplayQueue(
+            List<GuiMessage.Line> instance,
+            int i,
+            Object line,
+            Operation<Void> original,
+            @Local(name = "j") int index,
```

Comment:
How many integers are used? Perhaps it'd be wiser to use the field's ordinal.

While writing this comment, I've checked Mixin docs,

> Names of the local variable. This will not work on obfuscated code like Minecraft. For targeting unobfuscated code this is normally the least brittle option.

Can you convert both of the use cases to ordinals?

Reply:
Yup my bad, I forgot about this. (Though funnily enough I think using name here technically would work since those are the obfuscated names; obviously a bad practice though.)